docs/user-guide/k8s-install-and-config.md

```diff
@@ -0,0 +1,325 @@
+- or provision a Kubernetes cluster from popular Cloud vendors:
+  * [Amazon Elastic Kubernetes Service](https://aws.amazon.com/eks/)
+  * [Microsfot Azure Kubernetes Service](https://docs.microsoft.com/en-us/azure/aks/intro-kubernetes)
+  * [IBM Cloud Kubernetes Service](https://www.ibm.com/ca-en/cloud/kubernetes-service)
+  * [Google Cloud Kubernetes Engine](https://cloud.google.com/kubernetes-engine)
+
+### `kubectl` Tool
+
+You need `kubectl` CLI tool installed on your local computer where you want to manage the Kubernetes cluster. Please follow appropriate steps from official documentation [Install Tools](https://kubernetes.io/docs/tasks/tools/).
+
+## Downloading
+
+Downloading Zowe's container images manually is not required, since this can be done automatically when applying a Kubernetes [deployment configuration](#apply-zowe-core-components-and-start-zowe).
+
+However, it is possible to get the images without letting Kubernetes do the download. Using commands such as `docker pull` or `docker load` can allow you to get an image from a registry or attach an image you have downloaded directly. Zowe's container images can be found at:
+
+Registry: zowe-docker-release.jfrog.io
+Organization: ompzowe
+
+Full image addresses include,
+zowe-docker-release.jfrog.io/ompzowe/gateway-service:latest-ubuntu
+zowe-docker-release.jfrog.io/ompzowe/app-server:latest-ubuntu
+zowe-docker-release.jfrog.io/ompzowe/explorer-jes:latest-ubuntu
+
+## Installing
+
+You do not need to do any install task if using Zowe's Kubernetes configuration samples, it is automatic.
+
+An image could be considered "Installed" when it is findable by Kubernetes. Just like downloading, this is done automatically by Kubernetes but commands such as `docker pull` or `docker load` will accomplish the same task.
+
+## Configuring
+
+Zowe provides sample configurations that make it easy to get Zowe running in Kubernetes.
+You can use these directly, or as a reference.
+If you'd like to customize the configuration, note the following objects that are expected by the container deployments:
+
+|    Kind   | Name | Note |
+|-----------|------|------|
+| Namespace | zowe | |
+| ServiceAccount | zowe-sa | |
+| ConfigMap | zowe-certificates-cm | Contains zowe-certificates.env with the same format as seen on z/OS keystore |
+| Secret | zowe-certificates-secret | Contains the base64 PEM and P12 data for keystore and truststore |
+| Ingress | discovery-ingress | Used for external access to the Discovery service |
+| Ingress | gateway-ingress | Used for external access to the Gateway service |
+| Route | discovery | Used for external access to the Discovery service |
+| Route | gateway | Used for external access to the Gateway service |
+| Service | discovery-service | Used for external access to the Discovery service |
+| Service | gateway-service | Used for external access to the Gateway service |
+| PersistentVolumeClaim | zowe-workspace-pvc | |
```

Comment:
We have few more Kinds currently located in samples directory:

- [HorizontalPodAutoscaler](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/) in `containers/kubernetes/samples/horizontal-pod-autoscaler/`
- [PodDisruptionBudget](https://kubernetes.io/docs/tasks/run-application/configure-pdb/) in `containers/kubernetes/samples/pod-disruption-budget/`

Also the latest change we defined a new `ClusterIP` service for API Catalog in `containers/kubernetes/samples/api-catalog-service.yaml`.